unit-ruby.gemspec

```
@@ -35,14 +35,13 @@ Gem::Specification.new do |spec|
   spec.require_paths = ['lib']
 
   spec.add_dependency 'activesupport'
-  spec.add_dependency 'faraday', '~> 1.8.0'
-  spec.add_dependency 'faraday_middleware', '~> 1.0.0'
+  spec.add_dependency 'faraday', '>= 1.10', '< 3'
 
-  spec.add_development_dependency 'bundler', '~> 1.17'
   spec.add_development_dependency 'dotenv', '~> 2.7.6'
   spec.add_development_dependency 'pry'
   spec.add_development_dependency 'rake', '~> 13.0'
   spec.add_development_dependency 'rspec', '~> 3.0'
   spec.add_development_dependency 'rubocop', '~> 1.24.1'
+  spec.add_development_dependency 'faraday-retry'
```

Should this be in the non-dev dependencies?

```suggestion
  spec.add_dependency 'faraday-retry'
```